spring-cloud-stream-binder-kafka-core/src/main/java/org/springframework/cloud/stream/binder/kafka/properties/KafkaProducerProperties.java

```diff
@@ -26,8 +26,10 @@
  */
 public class KafkaProducerProperties {
 
+	@Deprecated
```

Comment:
It might be good to add a deprecation comment for what the suggested way is going forward.